en/flight_controller/nxp_mr_vmu_rt1176.md

```diff
@@ -0,0 +1,266 @@
+As a reference/evaluation design, it is intended to be replicated, modified or integrated in production volume by others.
+Several 3rd party manufacturers (such as Holybro.com) provide this or derivative commercial products.
+Note that NXP is a semiconductor vendor, and since it is a reference design, this particular board is not produced in high volume.
+
+This board takes advantage of multiple Pixhawk​​® open standards, such as the FMUv6X-RT Standard, [Autopilot Bus Standard](https://github.com/pixhawk/Pixhawk-Standards/blob/master/DS-010%20Pixhawk%20Autopilot%20Bus%20Standard.pdf), and [Connector Standard](https://github.com/pixhawk/Pixhawk-Standards/blob/master/DS-009%20Pixhawk%20Connector%20Standard.pdf).
+Equipped with a high performance NXP i.mx RT1176 dual core Processor, modular design, triple redundancy, temperature-controlled IMU board, isolated sensor domains, delivering incredible performance, reliability, and flexibility.
+
+:::tip
+This autopilot is [supported](../flight_controller/autopilot_pixhawk_standard.md) by the PX4 maintenance and test teams.
+:::
+
+## Introduction
+
+Inside the MR-VMU-RT1176, you can find an NXP i.MX RT1176, paired with sensor technology from Bosch®, InvenSense®, giving you flexibility and reliability for controlling any autonomous vehicle, suitable for both academic and commercial applications.
+
+The Pixhawk® 6X-RT's i.MX RT1176 Crossover dual-core MCU contain an Arm® Cortex®-M7 core running up to 1GHz and Arm® Cortex®-M4 core running up to 400MHz, has 2MB SRAM and external XIP Flash with 64MB.
+The PX4 Autopilot takes advantage of the increased processing power and RAM.
+Thanks to this increased processing power, developers can be more productive and efficient with their development work, allowing for complex algorithms and models.
+
+The FMUv6X-RT open standard includes high-performance, low-noise IMUs on board, designed for better stabilization.
+Triple-redundant IMU & double-redundant barometer on separate buses.
+When PX4 detects a sensor failure, the system seamlessly switches to another to maintain flight control reliability.
+
+An independent LDO powers every sensor set with independent power control.
+A vibration isolation System to filter out high-frequency vibration and reduce noise to ensure accurate readings, allowing vehicles to reach better overall flight performances.
+
+External sensor bus (SPI5) has two chip select lines and data-ready signals for additional sensors and payload with SPI-interface, and with an integrated Microchip Ethernet PHY, high-speed communication with mission computers via ethernet is now possible.
+
+The MR-VMU-RT1176 reference design is perfect for developers at corporate research labs, startups, academics (research, professors, students), and commercial application that like to try out T1 (2 wire) automotive ethernet.
+
```

Comment:
Moved this down here
```suggestion

Note that since it is a reference design, this _particular_ board is not produced in high volume.
Similar variants will be available from our licensees.

```